builder/ebs/builder_acc_test.go

```diff
@@ -2218,6 +2253,7 @@ build {
 	sources = ["amazon-ebs.windows-fastboot"]
 
 	provisioner "powershell" {
+		skip_clean = true
```

Comment:
Is this necessary? I know we discussed it out-of-band yesterday, but IIRC we didn't conclude this was the reason why the tests failed; if it is the reason though, I think it warrants a documentation update to highlight that it should be set for sysprep when using fast-boot.